packages/adaptive_scaffold/lib/adaptive_scaffold.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library adaptive_scaffold;
+
+/// A Calculator.
```

Comment:
Why are we checking in a package with no actual functionality? We should wait until we have at least some useful, real content before landing a package (and especially before *publishing* a package, which this PR will do currently.)

Reply:
Because of the size of the PR. It was recommended to split it into 3 PRs to make it easier to review. This PR would just hold boilerplate, the next would hold code/tests, and the last would hold the examples. I expect the PRs to be sent out in succession so it would be fully out before anyone could use it in some stable version. Is this not a good way to handle it?

Reply:
If it's going to be landed without any functionality, it should have `publish_to: none` in the pubspec until it's actually a useful package.

This seems like an odd cut-point to me though; there's almost nothing in this PR that requires actual review since it's almost all boilerplate, so I'm not sure how including initial functionality would make reviewing it harder. But if @dkwingsmt feels strongly about landing this now, it needs to not be published.

Reply:
I think just having the boilerplate at first makes it easier to drop the perceived changes significantly and makes the review process simpler for everyone involved. I've added the publish_to:none flag but if there's anything else you'd like me to do to make this more streamlined please let me know. Thanks.